<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompilerSettings</class>
 <widget class="QWidget" name="CompilerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QListWidget" name="compilerList">
       <property name="styleSheet">
        <string notr="true">font-size: 11pt;</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="moveUpButton">
         <property name="minimumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="go-up">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="moveDownButton">
         <property name="minimumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="go-down">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addCompilerButton">
         <property name="minimumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="list-add">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="deleteCompilerButton">
         <property name="minimumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="list-remove">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="compilerNameLabel">
       <property name="styleSheet">
        <string notr="true">font-size:11pt;font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Compiler Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="compilerName">
       <property name="styleSheet">
        <string notr="true">font-size:11pt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sourceExtensionsLabel">
       <property name="styleSheet">
        <string notr="true">font-size:11pt;font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Source Extensions</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sourceExtensions">
       <property name="styleSheet">
        <string notr="true">font-size:11pt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="advancedButton">
       <property name="text">
        <string>Advanced</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
